projects/igniteui-angular/src/lib/core/i18n/grid-resources.ts

```diff
@@ -277,6 +278,7 @@ export const GridResourceStringsEN: IGridResourceStrings = {
     igx_grid_pinning_uncheck_all_label: 'Unpin All',
     igx_grid_row_edit_btn_done: 'Done',
     igx_grid_row_edit_btn_cancel: 'Cancel',
+    igx_grid_row_edit_text: "You have {0} changes in this row and {1} hidden columns",
     igx_grid_toolbar_actions_filter_prompt: 'Filter columns list ...',
```

Comment:
```suggestion
    igx_grid_row_edit_text: 'You have {0} changes in this row and {1} hidden columns',
```
Single quotes to be consistent